resources/views/waitingrooms/index.blade.php

```diff
@@ -36,6 +37,11 @@
                             <td class="table__cell">{{ $room->campaign_run_id }}</td>
                             <td class="table__cell">{{ date('F d, Y', strtotime($room->signup_start_date)) }}</td>
                             <td class="table__cell">{{ date('F d, Y', strtotime($room->signup_end_date)) }}</td>
+                            @if (time() - (60 * 60 * 24) >= strtotime($room->signup_end_date))
```

Comment:
Pull down latest master, rebase with this branch and run `composer dump-autoload` and you should be able to access the helper functions.